### PR TITLE
Add assiginees to pull requests

### LINKED ISSUE #248: Add "assignees" to PRs and issues.
GitHub now supports multiple assignees. We would like to use that information to display it in our [pull-status](https://github.com/iphydf/pull-status) output.

### PR DESCRIPTION
**Pull request by GrayHatter**
closes #248

### CODE REVIEW

**Comment by GrayHatter** [comment]
@iphydf

File: src/GitHub/Data/Issues.hs [hunk 1]
```diff
@@ -24,6 +24,7 @@ data Issue = Issue
     , issueLabels      :: (Vector IssueLabel)
     , issueNumber      :: !Int
     , issueAssignee    :: !(Maybe SimpleUser)
```
Comment (phadej) [inline src/GitHub/Data/Issues.hs]: Remove `issueAssignee` field

File: src/GitHub/Data/Issues.hs [hunk 2]
```diff
@@ -162,22 +163,23 @@ instance FromJSON IssueComment where
...
-        <*> o .: "body"
-        <*> o .: "state"
-        <*> o .: "id"
-        <*> o .: "comments"
+        <*> o .:  "url"
+        <*> o .:  "created_at"
+        <*> o .:  "body"
+        <*> o .:  "state"
+        <*> o .:  "id"
+        <*> o .:  "comments"
```
Comment (phadej) [inline src/GitHub/Data/Issues.hs]: this style change is really unnecessary...

File: src/GitHub/Data/PullRequests.hs [hunk 3]
```diff
@@ -31,11 +31,11 @@ data SimplePullRequest = SimplePullRequest
     , simplePullRequestCreatedAt :: !UTCTime
     , simplePullRequestUser      :: !SimpleUser
     , simplePullRequestPatchUrl  :: !URL
-    , simpleIssueState     :: !IssueState
+    , simplePullRequestState     :: !IssueState
```
Comment (phadej) [inline src/GitHub/Data/PullRequests.hs]: hmm?

File: src/GitHub/Data/PullRequests.hs [hunk 4]
```diff
@@ -58,7 +58,9 @@ data PullRequest = PullRequest
     , pullRequestNumber         :: !Int
     , pullRequestHtmlUrl        :: !URL
     , pullRequestUpdatedAt      :: !UTCTime
-    , pullRequestBody           :: !Text
+    , pullRequestBody           :: !(Maybe Text)
+    , pullRequestAssignee       :: !(Maybe SimpleUser)
```
Comment (phadej) [inline src/GitHub/Data/PullRequests.hs]: remove this too

File: src/GitHub/Data/PullRequests.hs (hunk 3 above)
Comment (GrayHatter) [inline src/GitHub/Data/PullRequests.hs]: reverted, I was working from a dirty working dir, so I assumed it was a missed ctrl c/v

File: src/GitHub/Data/Issues.hs (hunk 1 above)
Comment (GrayHatter) [inline src/GitHub/Data/Issues.hs]: done

File: src/GitHub/Data/Issues.hs (hunk 2 above)
Comment (GrayHatter) [inline src/GitHub/Data/Issues.hs]: reverted

File: src/GitHub/Data/PullRequests.hs (hunk 4 above)
Comment (GrayHatter) [inline src/GitHub/Data/PullRequests.hs]: done

**Comment by GrayHatter** [comment]
@phadej no idea how to fix that last test. Sorry mate!

**Comment by phadej** [comment]
@GrayHatter I'll check this next week, and try to wrap up the release. Thanks!

**Comment by GrayHatter** [comment]
@phadej thank you sir, ping me if you need anything else!

**Comment by iphydf** [comment]
Any news here?

**Comment by phadej** [comment]
I'm investigating one mailfunction in our internal service. After that I'll go thru prs and issues and hopefully cut 0.15.0 today.

**Comment by phadej** [comment]
done in #251

**Pull request closed without merging**